data/json/monsters/defense_bot.json

```diff
@@ -117,5 +117,71 @@
     "death_drops": { "groups": [ [ "robots", 4 ], [ "skitterbot", 1 ] ] },
     "death_function": [ "BROKEN" ],
     "flags": [ "SEES", "HEARS", "BASHES", "GOODHEARING", "ELECTRONIC", "NO_BREATHE", "PRIORITIZE_TARGETS", "PATH_AVOID_DANGER_2" ]
+  },
+  {
+    "id": "mon_dispatch",
+    "type": "MONSTER",
+    "name": "riot dispatch",
+    "description": "The Northrop Dispatch, designed for crowd control situations, carries scrap parts and explosive payloads, which it rapidly assembles into kamikaze drones depending on what its AI deems necessary.  The bright green-and-yellow paint marks a riot control variant, specializing in (but not restricted to) nonlethal force.",
```

Comment:
I had missed this until now since I didn't have time to review this thoroughly.  This is not ok lore-wise in DDA, it needs to simply have a payload of drones and bombs. It's also perplexing why it is even here, as it's far, far more reasonable to just say that the hacks are stored in an assembled state on-board, but have a mechanism to non-offensively self-destruct them to prevent capture.